### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by 4c74356b41**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

**Comment by PRmerger** [comment]
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @azdocprs, has been notified to review your proposed change.

**Comment by tysonn** [comment]
@MikeWasson - Mike, could you review this proposed update, please?

File: includes/guidance-compute-single-vm-windows.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ Provisioning VM in Azure involves more moving parts than just the VM itself. The
 
 - Your VM may be affected by [planned maintenance][planned-maintenance] or [unplanned maintenance][manage-vm-availability]. You can use [VM reboot logs][reboot-logs] to determine whether a VM reboot was caused by planned maintenance.
 
-- VHDs are backed by [Azure Storage][azure-storage], which is replicated for durability and availability.
+- VHDs are backed up by [Azure Storage][azure-storage], which is replicated for durability and availability.
```
Comment (MikeWasson) [inline includes/guidance-compute-single-vm-windows.md]: "backed up" would mean a copy, "backed" is actually correct here (Azure Storage is the backing storage for the VHD)

**Comment by MikeWasson** [comment]
One comment. The rest look good, thanks!

**Comment by 4c74356b41** [comment]
i think i reverted it

**Comment by MikeWasson** [comment]
Looks good

**Comment by 4c74356b41** [comment]
@MikeWasson thanks
@tysonn Yeah, I'm sorry to be such an pain in the neck, but could you perhaps look into other PR's i got (plenty of them, tbh), some of them are like 3 weeks old already?

**Comment by MikeWasson** [comment]
#sign-off

**Comment by PRmerger** [comment]
Only the author of the article can use the sign off comment to add the ready-to-merge label.

**Pull request merged**